<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Programmer</class>
 <widget class="QMainWindow" name="Programmer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>820</width>
    <height>650</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>等线</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>DLMS Programmer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Paint.ico</normaloff>Paint.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="AddressLogic">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="SerialNo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>逻辑地址</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>物理起始</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Baudrate">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <item>
     <property name="text">
      <string>1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>自动</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="CheckBit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>无校验</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>奇校验</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>偶校验</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="StopBit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>1位</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1位半</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2位</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="Password">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>350</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>350</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>350</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>3030303030303030</string>
    </property>
    <property name="maxLength">
     <number>40</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="AddressStart">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="EncryptKey">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>350</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>350</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>350</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>30303030303030303030303030303030</string>
    </property>
    <property name="maxLength">
     <number>32</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>加密密钥</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>100</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>认证密钥</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Level">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>无认证</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>低等级</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>高等级</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="AuthKey">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>350</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>350</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>350</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>30303030303030303030303030303030</string>
    </property>
    <property name="maxLength">
     <number>32</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>验证密码</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="LOG">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>560</width>
      <height>410</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>160</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>800</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>串口选择</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>校验位</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>停止位</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>认证等级</string>
    </property>
   </widget>
   <widget class="Line" name="line_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>821</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>821</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="AddressMode">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <item>
     <property name="text">
      <string>1字节</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2字节</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4字节</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>464</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>地址模式</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AddressClient">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>16</string>
    </property>
    <property name="maxLength">
     <number>3</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>客户地址</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AddressEnd">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>物理结束</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonLoad">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>190</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>未加载</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonStart">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>190</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>开始运行</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="BoxSet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>是否设置</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="BoxGet">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>是否检查</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Result">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>230</y>
      <width>231</width>
      <height>410</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>160</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>800</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="Port">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>5000</string>
    </property>
    <property name="maxLength">
     <number>6</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="IPAddress">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>192.168.100.1</string>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string> 服务器地址</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>SerialNo</tabstop>
  <tabstop>Baudrate</tabstop>
  <tabstop>CheckBit</tabstop>
  <tabstop>StopBit</tabstop>
  <tabstop>IPAddress</tabstop>
  <tabstop>Port</tabstop>
  <tabstop>AddressClient</tabstop>
  <tabstop>AddressMode</tabstop>
  <tabstop>AddressLogic</tabstop>
  <tabstop>AddressStart</tabstop>
  <tabstop>AddressEnd</tabstop>
  <tabstop>Level</tabstop>
  <tabstop>EncryptKey</tabstop>
  <tabstop>Password</tabstop>
  <tabstop>AuthKey</tabstop>
  <tabstop>BoxSet</tabstop>
  <tabstop>BoxGet</tabstop>
  <tabstop>ButtonLoad</tabstop>
  <tabstop>ButtonStart</tabstop>
  <tabstop>LOG</tabstop>
  <tabstop>Result</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
